common/src/test/scala/weco/concepts/common/fixtures/ElasticsearchResponses.scala

```diff
@@ -0,0 +1,179 @@
+        Success(
+          HttpResponse(
+            status = StatusCodes.OK,
+            entity = HttpEntity(
+              ContentTypes.`application/json`,
+              ujson.write(
+                ujson.Obj(
+                  "succeeded" -> true,
+                  "num_freed" -> 1
+                )
+              )
+            )
+          )
+        )
+      )
+  }
+
+  def handleSearch[T: Indexable](
+    docs: Seq[T],
+    transformPit: String => String = identity
+  )(implicit
+    mat: Materializer
+  ): PartialFunction[HttpRequest, Future[Try[HttpResponse]]] = {
+    case HttpRequest(HttpMethods.POST, uri, _, entity, _)
+        if uri == Uri("/_search") =>
+      implicit val ec: ExecutionContext = mat.executionContext
+      Unmarshal(entity).to[String].map(ujson.read(_)).map { requestBody =>
+        val pit = transformPit(requestBody("pit")("id").str)
+        val searchAfter =
+          requestBody
+            .opt[Seq[Int]]("search_after")
+            .flatMap(_.headOption)
+            .map(_ + 1) // Because we're slicing using the indices below
+            .getOrElse(0)
+        val size = requestBody("size").num.toInt
+        Success(
+          HttpResponse(
+            status = StatusCodes.OK,
+            entity = HttpEntity(
+              ContentTypes.`application/json`,
+              ujson.write(
+                ujson.Obj(
+                  "pit_id" -> pit,
+                  "hits" -> ujson.Obj(
+                    "hits" ->
+                      docs.zipWithIndex
+                        .slice(
+                          searchAfter,
+                          searchAfter + size
+                        )
+                        .map { case (doc, idx) =>
+                          ujson.Obj(
+                            "_source" -> doc.toDoc,
+                            "sort" -> Seq(idx)
+                          )
+                        }
+                  )
+                )
+              )
+            )
+          )
+        )
+      }
+  }
+
+  def mgetResponse[T: Indexable](
+    index: String,
+    existing: Seq[T],
+    entity: HttpEntity
+  )(implicit mat: Materializer): Future[Try[HttpResponse]] = {
+    implicit val ec: ExecutionContext = mat.executionContext
+    Unmarshal(entity).to[String].map(ujson.read(_)).map { requestBody =>
+      val requestedIds = requestBody("ids").arr.map(_.str)
+      val existingMap = existing.map(doc => doc.id -> doc).toMap
+      val docs = requestedIds
+        .map(id =>
+          existingMap
+            .get(id)
+            .map(indexable =>
+              ujson.Obj(
+                "_index" -> index,
+                "_id" -> id,
+                "_version" -> 1,
+                "found" -> true,
+                "_source" -> indexable.toDoc
+              )
+            )
+            .getOrElse(
+              ujson.Obj("_index" -> index, "_id" -> id, "found" -> false)
+            )
+        )
+      Success(
```

Comment:
This `Success(HttpResponse(...))` pattern seems quite common. Is there any value in wrapping it?  Save a few parentheses.

Reply:
Think I would prefer to keep the standard library classes as it's mostly used in test fixture code